### DISCUSSION TITLE
How to set BorderColor?

### DISCUSSION THREAD

**Discussion by FZliweiliang** [dlemstra/magick-wasm#83] [Q&A]
Didn't see the official BorderColor demo set up

**Comment by dlemstra** [comment]
The image has a `borderColor` property? Is that what you are looking for?

**Reply by FZliweiliang** [reply]
Yes, but my call to borderColor has no effect, is there a demo?

` 
ImageMagick.read(fs.readFileSync(input), (image) => {

            if (bordercolor) {

              const { rgba } = parse(bordercolor);

              image.borderColor = new MagickColor(

                rgba[0],
                rgba[1],
                rgba[2],
                rgba[3]
              );
            }
            
            image.write(format, (data) => {
              if (to) fs.writeFileSync(to, data);
            });
})

`

**Reply by dlemstra** [reply]
What do you think that it does? It is something that is used by other operations and just setting the property does nothing.

**Reply by FZliweiliang** [reply]
Used to modify the stroke color when calling the border api

**Reply by dlemstra** [reply]
I just added a unit test that does this:

```js
image.borderColor = MagickColors.Purple;
image.border(1, 1);
```

And the border that is added is purple. How are you testing this?
